packages/cysync-core/src/pages/MainApp/Components/ContactSupport.tsx

```diff
@@ -0,0 +1,318 @@
+import {
+  Button,
+  CheckBox,
+  DialogBox,
+  DialogBoxFooter,
+  DialogBoxHeader,
+  Flex,
+  Input,
+  LangDisplay,
+  TextAreaInput,
+  TextareaVector,
+  Typography,
+  Divider,
+  Container,
+  DeviceNotConnected,
+  Close,
+  Throbber,
+  Dropdown,
+  useTheme,
+} from '@cypherock/cysync-ui';
+import React, { FC, useState } from 'react';
+import { styled } from 'styled-components';
+
+interface DataItemType {
+  id: string;
+  text: string;
+}
+
+interface ContactSupportProps {
+  header?: string;
+  title?: string;
+  subTitle?: string;
+  btnCancel: string;
+  btnSubmit: string;
+  email?: string;
+  category?: string;
+  description?: string;
+  label1?: string;
+  label2?: string;
+  handleApplicationCheckedItem: () => void;
+  handleDeviceCheckedItem: () => void;
+  isApplicationChecked: boolean;
+  isDeviceChecked: boolean;
+  error?: boolean;
+  deviceNotConnected?: boolean;
+  errorMsg?: string;
+  connectionErrorMsg?: string;
+  deviceErrorMsg?: string;
+  deviceError?: boolean;
+  confirmMsg?: string;
+  logMsg?: string;
+  globalError?: string;
+  data: DataItemType[];
+}
+
+const CustomFlex = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 16px;
+  @media ${({ theme }) => theme.screensHeight.lg} {
```

Comment:
If I give min-height it works.. but when I use like this.. it is not working.. can you please check why